### PR TITLE
An idempotent provisioning

### PR DESCRIPTION
**Pull request by shurph**
I think it would be good to have ability to retry a provisioning safely.
This PR is fixed the issues with a PG user creation and with a `.bashrc` provisioning.

btw, there are still few issues in the https://github.com/wagtail/bakerydemo repo. I'll try to fix them soon.

### CODE REVIEW

File: vagrant/provision.sh [hunk 1]
```diff
@@ -29,18 +31,44 @@ apt-get install -y libenchant-dev
...
-source /usr/local/bin/virtualenvwrapper.sh
-EOF
-
+# just put these three values to .bashrc:
+BASHRC_LINE_1="export WORKON_HOME=/home/vagrant/.virtualenvsv"
+BASHRC_LINE_2="export VIRTUALENVWRAPPER_PYTHON=python3"
+BASHRC_LINE_VENV="source /usr/local/bin/virtualenvwrapper.sh"
+IS_NEED_UPDATE_BASHRC_VENV=no
+
+for i in $(seq 1 2);
```
[the remark this answers is not in the document]
Comment (shurph) [reply vagrant/provision.sh]: Hi @zerolab, I fixed modified the PR according your suggestions.
How will be better: leave it as separate commit or "amend" it to a previous?

**Comment by zerolab** [comment]
LGTM :shipit:

**Comment by shurph** [comment]
sure, will do

**Comment by shurph** [comment]
@gasman I've fixed the issue with a fresh installation. It was just my wrong assumption about psql :-(

**Comment by gasman** [comment]
Thanks @shurph!

**Pull request merged**